core/src/main/scala/org/apache/spark/util/AkkaUtils.scala

```diff
@@ -147,7 +146,7 @@ private[spark] object AkkaUtils extends Logging {
   def askWithReply[T](
       message: Any,
       actor: ActorRef,
-      timeout: FiniteDuration): T = {
+      timeout: RpcTimeout): T = {
```

Comment:
hmm, I disagree.  No harm in making these changes.  If we're really certain `AkkaUtils` wont' be used anymore, then we'll delete it ... but in the meantime, these changes are here just in case.

Maybe it wasn't worth the effort in the first place (my fault!) but I dont' see the harm ...